kv/kv.go

```diff
@@ -92,8 +92,6 @@ var (
 type RetrieverMutator interface {
 	Retriever
 	Mutator
-	// set allowed options of current operation in each TiKV disk usage level.
-	SetDiskFullOpt(level kvrpcpb.DiskFullOpt)
```

Comment:
If this interface is removed, then there are some other interface need to be removed:
One is MemBuffer::SetDiskFullOpt, Another is client-go MemDB::SetDiskFullOpt.

Reply:
got, I'll move the goclient later